CONTRIBUTING.md

```diff
@@ -510,7 +510,7 @@ An entry into the changelog is required for the following reasons:
 - Changes to default settings
 - New components being added
 
-It is reasonable to omit an entry to the changelog under these circuimstances:
+It is reasonable to omit an entry to the changelog under these circumstances:
```

Comment:
Can you please open a separate PR for this typo?

Reply:
I mean, yes. But that seems a bit extra?

Reply:
Removed it from this PR

Reply:
That’s just how it’s done for this repo.